### DISCUSSION TITLE
Connecting from browser to dGraph wss

### DISCUSSION THREAD

**Discussion by gotjoshua** [enisdenjo/graphql-ws#313] [Q&A] [answered]
i am trying to use the observable recipe with a dgraph GraphQL backend.

Interestingly enough, i can connect to a subscription  query via GraphQL Playground
but not from my localhost app in the browser.

Actually, polling will be performant enough for my current use case, but I want to know how to set up wss with server push, for future reference...

## browser
![errorsBrowser](https://user-images.githubusercontent.com/8686454/154865638-027f927e-bde2-40e6-b999-773383b6bbbc.png)
![requestHeadersBrowser](https://user-images.githubusercontent.com/8686454/154865720-054049ec-aac6-41d9-a069-564b064996f9.png)

## GraphQL Playground
![successHeadersGraphQLPlayground](https://user-images.githubusercontent.com/8686454/154865769-1a3a7a4a-b8a8-401e-95e5-6670393e5020.png)

the error doesn't tell me much...

any help?

**Comment by enisdenjo** [comment, marked as answer]
Probably because Dgraph still does not support `graphql-ws`. [GraphQL playground still uses the legacy `subscription-transport-ws`](https://github.com/graphql/graphql-playground/issues/1291) which is why you can connect using it.

[Made an issue at Dgraph's Discuss a while ago.](https://discuss.dgraph.io/t/support-graphql-over-websocket/12698/15)
